### PR TITLE
Front 2490: Table pattern

### PR DESCRIPTION
**Pull request by Maxfire**
Jira issue(s):
- https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-426

### CODE REVIEW

**Review by abel-santos-corral — DISMISSED** [review]
Review finished. Only added +1 row at table.ui_patterns.yml and changed the content of the three rows of the preview.

File: templates/overrides/views/views-view-table.html.twig [hunk 1]
```diff
@@ -4,97 +4,21 @@
...
+      }]) %}
     {% endfor %}
-    </tbody>
-  </table>
-</div>
+  {% endfor %}
+  {% set rows_items = rows_items|merge([{'cells': row_content}]) %}
+{% endfor %}
+{{ pattern('table', {
+  caption: caption,
```
Comment (escuriola) [inline templates/overrides/views/views-view-table.html.twig:36]: Missing table_responsive:
```suggestion
  caption: caption,
  table_responsive:  responsive ? true...
```

File: templates/overrides/views/views-view-table.html.twig [hunk 2]
```diff
@@ -4,97 +4,21 @@
...
+  rows: rows_items,
+  table_head: table_head,
+}) }}
```
Comment (escuriola) [inline templates/overrides/views/views-view-table.html.twig:40]: Details (summary and description) also missing inside caption.

File: templates/overrides/dataset/table.html.twig [hunk 3]
```diff
@@ -5,79 +5,9 @@
...
-          </{{ cell.tag }}>
-        {% endfor %}
-        </tr>
-      {% endfor %}
-      </tfoot>
-    {% endif %}
-  </table>
-</div>
+{{ pattern('table', {
+  caption: caption,
```
Comment (escuriola) [inline templates/overrides/dataset/table.html.twig:9]:
```suggestion
  caption: caption,
   table_responsive: true,
```

**Review by escuriola — CHANGES_REQUESTED** [review]

File: templates/overrides/views/views-view-table.html.twig (hunk 2 above)
Comment (escuriola) [reply templates/overrides/views/views-view-table.html.twig:40]: Checking with "seven" theme, seems to be a collapsible field. The summary is the link and the description the content. 
image

File: templates/overrides/views/views-view-table.html.twig (hunk 1 above)
Comment (Maxfire) [reply templates/overrides/views/views-view-table.html.twig:36]: As we checked it's already in place.

File: templates/overrides/dataset/table.html.twig (hunk 3 above)
Comment (Maxfire) [reply templates/overrides/dataset/table.html.twig:9]: same as before, it's already in place.

File: templates/overrides/views/views-view-table.html.twig (hunk 2 above)
Comment (Maxfire) [reply templates/overrides/views/views-view-table.html.twig:40]: Added in the view template.

**Review by drishu — APPROVED** [review]

**Review by escuriola — APPROVED** [review]

**Pull request merged**